index.html

```diff
@@ -7,6 +7,8 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta content="text/html; charset=utf-8" http-equiv="Content-Type">
     <meta content="noindex" name="robots">
+    <meta property="og:image" content="http://cs.smu.ca/~ka_kashif/SMU-Hack-A-Week-1/images/HackAWeektwitter.jpg">
```

Comment:
Yes I forgot to change that. I'll do that soon.
On Apr 26, 2015 4:59 PM, "Joel Kuntz" <notifications@github.com> wrote:

> In index.html
> <https://github.com/SMUhackaweek/SMU-Hack-A-Week/pull/25#discussion_r29113208>
> :
>
> > @@ -7,6 +7,8 @@
> >      <meta name="viewport" content="width=device-width, initial-scale=1">
> >      <meta content="text/html; charset=utf-8" http-equiv="Content-Type">
> >      <meta content="noindex" name="robots">
> > +    <meta property="og:image" content="http://cs.smu.ca/~ka_kashif/SMU-Hack-A-Week-1/images/HackAWeektwitter.jpg">
>
> Should this be a link to the proper username that is serving the website?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SMUhackaweek/SMU-Hack-A-Week/pull/25/files#r29113208>.
>